CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/en.lproj/Localizable.strings

```diff
@@ -36,7 +36,14 @@
 "ginicapture.camera.documentValidationError.tooManyPages" = "The document can only have a maximum of 10 pages.";
 "ginicapture.camera.documentValidationError.wrongFormat" = "Only files of the formats jpg, png, gif or pdf can be analysed";
 "ginicapture.camera.fileImportTip" = "Now documents can be conveniently uploaded for analysis.";
-"ginicapture.camera.fileImportButtonLabel" = "Import";
+"ginicapture.camera.fileImportButtonLabel" = "Files";
+"ginicapture.camera.flashButtonLabel.On.Voice.Over" = "Flash On";
+"ginicapture.camera.flashButtonLabel.Off.Voice.Over" = "Flash Off";
+"ginicapture.camera.flashButtonLabel.On" = "On";
+"ginicapture.camera.flashButtonLabel.Off" = "Off";
+"ginicapture.camera.thumbnail" = "Photo review";
```

Comment:
[Clarification] Do we need this label ginicapture.camera.thumbnail?
I don't see it in the design